modules/ROOT/pages/static-ip-config.adoc

```diff
@@ -0,0 +1,40 @@
+:experimental:
+= Configuring FCOS to Use a Static IP Address
+By default, an FCOS instance will attempt to grab a DHCP address from the local network. However, if you need FCOS to use a static IP address, you can do so by specifying the NetworkManager configuration in the Ignition configuration file.
+
+As with any custom configuration on FCOS, you can write specific files in the xref:ign-storage.adoc[`storage` node] of the Ignition file.
+
+The following snippet shows how to assign the following to eth0:
+
+* static IP range: `192.0.2.1/24`
+* gateway: `192.0.2.255`
```

Comment:
Addresses that end in `255` are typically broadcast addresses and wouldn't be used as a gateway.  I'd suggest changing the value of the static IP to `192.0.2.10/241 and the gateway to `192.0.2.1`